### DISCUSSION TITLE
Main Window Transparent Back Ground

### DISCUSSION THREAD

**Discussion by RyanMGallagher1** [AntonyCorbett/OnlyT#473] [Ideas]
Suggestion: Make background of the Main Timer Transparent like the countdown timer.  We would use this on a confidence monitor in the back of the hall. We then could display the timer of the same monitor. Currently the only way to do it is to have 2 dedicated monitors. One for the confidence monitor for the speaker and 1 for the OnlyT Timer.

Would be a great enhancement.

Thank you for the work you have put into this project. Thousands are truly benefiting from it.

**Comment by AntonyCorbett** [comment]
@RyanMGallagher1 Moved to Issue #477
